jetstream/analysis.py

```diff
@@ -41,6 +44,9 @@
 
 from . import bq_normalize_name
 
+# if TYPE_CHECKING:
+#     from mozanalysis.metrics import DataSource
+
```

Comment:
```suggestion
```
Can we remove this?

Reply:
Ok I went ahead and updated the logic a little so that the sanity metrics are not computed discretely, but they *are* computed and they *are* included in the views. I checked with DS and it sounded like these metrics could be useful if we made them more visible, so figured we should not omit them or make them *less* visible. We can follow up later on how best to make them more visible.